utils/sql/git/required/2021_06_17_server_first_levels.sql

```diff
@@ -0,0 +1,14 @@
+-- ----------------------------
+-- Table structure for server_first_levels
+-- ----------------------------
+CREATE TABLE `server_first_levels` (
```

Comment:
There's no primary key on this, maybe add an `id` column and use `character_id` over character name.

We can translate to character name on the server side and it allows us to maybe inner join to this table a lot cleaner because we can just compare ID to ID without looking up ID from the name..